### DISCUSSION TITLE
Should we mention the urql/introspection codegen plugin?

### DISCUSSION THREAD

**Discussion by frederikhors** [urql-graphql/urql#1899] [Ideas]
Should we mention on the [schema awareness page](https://formidable.com/open-source/urql/docs/graphcache/schema-awareness/) that the Codegen team has created a dedicated plugin for urql?

https://www.graphql-code-generator.com/docs/plugins/urql-introspection

**Comment by bartenra** [comment]
Is it good? What does it do exactly?

**Reply by frederikhors** [reply]
Read docs.

**Comment by kitten** [comment]
No, because the appropriate page for GCg doesn't exist yet in the docs in the first place. Once that story's a good fit combined with fragments and so on the page may be written at which point this would become integrated anyway

**Reply by frederikhors** [reply]
Ok, but I'm using it day by day and it works amazingly!
